### DISCUSSION TITLE
3章15節「HTMLのフォーム」練習問題のコーディング

### DISCUSSION THREAD

**Discussion by satsukizzz** [satsukizzz/n-playspace-http#15] [Show and tell]
コーディングするという方、募集しています 🤚

**Comment by HappyKumamoto** [comment]
今、作成中のがあるので、出させていただきたいです！

ただ、去年のコンテストに出した、自分のWebページの、クイズの正解が出たら
先に進んで、応募またはアンケートフォームという形なので、
まるっきり、方向性が決まってしまうので、いいのかな、こんなのでという心配があります。

このクイズの続きになります。N予備校の部分の代わりとして作っています。
先々、pugになったら、第一弾と第二弾用にも分けることができるのでと思いつつ
↓
https://happykumamoto.github.io/tastyRed/

採用されなくてもいいので、とりあえず作成してみます。

**Reply by HappyKumamoto** [reply]
いや、申し訳ないです。
これ、忘れてください。
授業の通りに作成されているのに、全くわかっていませんでした。

授業、テキストどおりにさせてもらいます。🙇‍♀️

**Reply by satsukizzz** [reply]
いったん、お互いにどうやってプルリクを上げたりレビューしたりするか、というのを練習してもらおうかと思ってます！
まずはテキスト通りでお願いします 👍

**Reply by HappyKumamoto** [reply]
了解しました！😁💦

**Comment by HappyKumamoto** [comment]
こちらもキャンセルさせてください。
誠に申し訳ありません。🙇
